app/controllers/api/v1/authentication_controller.rb

```diff
@@ -12,8 +12,7 @@ def create
 
         raise Errors::AuthenticateError unless user.authenticate(params.require(:password))
 
-        render json: UserSerializer.render_as_json(user, root: :user, view: :with_auth_token),
-          status: :created
+        respond_with_json(UserSerializer.render_as_json(user, root: :user, view: :with_auth_token), :created)
```

Comment:
Shouldnt this be `render_with_success`? Also we need a notice here.

Reply:
Apis developed earlier. need to change accordingly.